### ISSUE TITLE
Separate the tool files from project files

### ISSUE THREAD

**Issue by lkjimy** [nickberens360/atomic-docs#50]
I'm a Front-End developer, and got to say that this tool awesome!

To organize the project, i would put the tool inside it's own folder, therefore avoiding mixing my project files with files from tools. It makes updating the tool a lot more easy and safe. For that would be necessary a file to carry the user configurations.

It's possible to make this kind of setup today?

**Comment by nickberens360** [comment]
@lkjimy Yes that's been on my todo list! I'll update here when completed.
